members/B001267.yaml

```diff
@@ -127,15 +127,14 @@ contact_form:
             VET.AFFRS: Veterans
             NAT.SEC: National Security
             WAS.REF: Washington Reform
-            OTHER: Other
```

Comment:
why was this causing the problem?

Reply:
"Other" was being selected as the topic option for each message, thus causing an error as the target's form didn't have that option available anymore. Deleting the option from the yaml (and updating the target's topic mapping in Artemis) ensure yamltron does not select "Other" as topic going forward.